docs/asf-docs/Introduction_hyp3_mintpy.ipynb

```diff
@@ -0,0 +1,527 @@
+    "# clip off files\n",
+    "cutGeotiffsByLine(files)    "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "2.3.3 Copy metadata file and dem file to DEM subdirectory"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# copy the *.txt and dem.tif in one subdirctory to DEM subdirectory\n",
+    "os.system(\"mkdir -p {}/DEM {}/inputs\".format(outdir, outdir))\n",
+    "prefix = filelist1[-1].split('.zip')[0]\n",
+    "fileprefix = os.path.basename(prefix)\n",
+    "txtfile = \"{dir}/{fileprefix}.txt\".format(dir = prefix, fileprefix = fileprefix)\n",
+    "demfile = \"{dir}/{fileprefix}_dem_clip.tif\".format(dir = prefix, fileprefix = fileprefix)\n",
+    "\n",
+    "cmd1 = \"cp {file} {data_dir}/DEM/dem_clip.txt\".format(file = txtfile, data_dir = outdir)\n",
+    "os.system(cmd1)\n",
+    "cmd2 = \"cp {file} {data_dir}/DEM/dem_clip.tif\".format(file = demfile, data_dir = outdir)    \n",
+    "os.system(cmd2)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"prepared data are at {}\".format(outdir))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Prepared data structure are:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import subprocess\n",
+    "process = subprocess.run(\"tree {}\".format(outdir), shell=True, check=True, stdout=subprocess.PIPE, universal_newlines=True)\n",
+    "output=process.stdout\n",
+    "print(output)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "2.4 Run Time-series Analysis application\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from mintpy import smallbaselineApp\n",
+    "argv =[]\n",
+    "argv.append(config_file)\n",
+    "smallbaselineApp.main(argv)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "2.5 Display the analysis results\n",
+    "\n",
+    "There are a few scripts used to display the analysis results. There are in the MINTPY_HOME/mintpy. Here we show two majoy disaply scripts."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "2.5.1 Disaply velocity of displacement\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from mintpy import view\n",
+    "file = \"velocity.h5\"\n",
+    "argv =[]\n",
+    "argv.append(file)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "%matplotlib notebook\n",
+    "view.main(argv)"
```

Comment:
I don't see a need for two cells here, I think it's just:
```
%matplotlib notebook
from mintpy import view
view.main(['velocity.h5'])
```
Or even:
```
%matplotlib notebook
!view.py velocity.h5
```

Reply:
changed